Xamarin.Forms.Platform.Android/CollectionView/ItemsViewAdapter.cs

```diff
@@ -65,10 +79,21 @@ public override RecyclerView.ViewHolder OnCreateViewHolder(ViewGroup parent, int
 				return new TextViewHolder(view);
 			}
 
-			var itemContentView = new ItemContentView(parent.Context);
+			var itemContentView = CreateItemContentView(parent.Context);
 			return new TemplatedItemViewHolder(itemContentView, ItemsView.ItemTemplate);
 		}
 
+		ItemContentView CreateItemContentView(Context context)
+		{
+			if(ItemsView.ItemSizingStrategy == ItemSizingStrategy.MeasureAllItems
+				|| _size == null)
+			{
+				return new ItemContentView(context);
+			}
+
+			return new ItemContentView(context);
+		}
```

Comment:
What is the meaning of this method if it always returns `new ItemContentView(context)`?

Reply:
Leftover from an earlier implementation. I'll remove it.